conf/config.toml

```diff
@@ -17,6 +17,19 @@ initial-cluster-state = "new"
 lease = 3
 tso-save-interval = "3s"
 
+# Path of file that contains list of trusted SSL CAs.
+tls-ca-path = ""
+# Path of file that contains X509 certificate in PEM format.
+tls-cert-path = ""
+# Path of file that contains X509 key in PEM format.
+tls-key-path = ""
+# enable client certificate auth, if true following two settings shouldn't be empty
+client-cert-auth = false
```

Comment:
As follower pd-servers redirect all requests to leader, so typically if if TLS is enabled on server side, client side must enable it too. Therefore I think this flag looks redundant to me.

Reply:
@disksing @huachaohuang 

Maybe we should let the client redirect to the new leader to reduce the latency.